db/migrate/20200219160002_create_emails.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEmails < ActiveRecord::Migration[6.0]
+  def change
+    create_table :emails do |t|
+      t.string :to, null: false
```

Comment:
I'm wondering if it's worth associating emails to the application form they pertain to... the only emails which I can think of that wouldn't have an application form to attach to are sign up emails. Then again, my hunch is this feature isn't that interesting for sign in/up emails, but for transactional emails between users/referees/providers which can all be tied to an application form.

Reply:
The problem is that we don't know at the moment, we'd have to update every email to try and add that information. Perhaps we should do that anyway, to add the `reference` to every email.

What I was thinking is that we could link email addresses to this page, which then shows all emails to a certain user.